examples/source/1.components/amp-script/index.html

```diff
@@ -242,5 +249,38 @@
         span.textContent = getOS();
       </script>
     </div>
+
+    <!--
+        ## Personalization
+        Similarly, you can use the `navigator` object, or other means, to personalize content for your user. The following script detects the browser's language and displays a localized greeting.
+    -->
+    <div>
+      <amp-script layout="fixed-height"
+        height="40"
+        script="translation-script"
+        class="sample">
+        <h2 id="translated-greeting">Saluton</h2>
+      </amp-script>
+  
+      <script id="translation-script" type="text/plain" target="amp-script">
+        const translationMap = {
+          'en': 'Hello',
+          'fr': 'Bonjour',
+          'es': 'Hola',
+          'hi': 'हैलो',
+          'zh': '你好',
+          'pr': 'Olá'
+        };
+  
+        const lang = navigator.language.slice(0, 2);
+        const translation = translationMap[lang];
+  
+        if (translation) {
+          let greeting = document.getElementById('translated-greeting');
+          greeting.innerHTML = translation;
+        }
```

Comment:
nit: show a message if language not supported to avoid confusion

Reply:
I chose to make the default message in Esperanto. I suppose no one's going to pick up on that one 😉

Reply:
See my comment below. The current implementation visible switches from Esperanto to a different language which is a flicker effect we should avoid.

Reply:
Yep. Just removed the default text from this sample and from another.